contributing/community/review-comments.rst

```diff
@@ -0,0 +1,177 @@
+constructive, respectful and helpful reviews and replies.
+
+.. tip::
+
+    This guide is not about lecturing you to "conform" or give-up
+    your ideas and opinions but helping you to better communicate,
+    prevent possible confusion, and keeping the Symfony community a
+    welcoming place for everyone. **You are free to disagree with
+    someone's opinions, just don't be disrespectful.**
+
+First of, accept that many programming decisions are opinions.
+Discuss trade offs, which you prefer, and reach a resolution quickly.
+It's not about being right or wrong, but using what works.
+
+Tone of voice
+-------------
+
+We don't expect you to be completely formal, or to even write error-free
+English. Just remember one thing: Don't swear, and be respectful to others.
+
+Don't reply in anger or with an aggressive tone. You're angry, we understand
+that, but swearing/cursing and name calling doesn't really encourage anyone to
+help you. Take a deep breath, count to 10 and try to *clearly* explain what problems
+you encounter.
+
+Gender pronouns
+---------------
+
+While not "formally" required it's better to use gender-neutral pronouns.
+Unless someone "indicated" their pronouns, use "they", "them" instead of
+"he", "she", "his", "hers", "his/hers", "he/she", etc.
+
+Try to avoid using wording that may be considered excluding and needlessly gendered,
+like for example words that have a male base. For example we recommend to use other
+words like “folks”, “team”, “everyone” in place of “guys”.
+
+Giving positive feedback
+------------------------
+
+While reviewing issues and pull requests you may run into some suggestions
+(including patches) that don't reflect your ideas, are not good, or downright wrong.
+
+Now, when you prepare your comment, consider the amount of work and time the author
+has spent on their idea and how your response would make them feel;
+
+Did you correctly understand their intention? Or are you making assumptions?
+Which ever you respond, be explicit. Remember people don't always understand your
+intentions online.
+
+Avoid using terms that could be seen as referring to personal traits ("dumb", "stupid").
+Assume everyone is intelligent and well-meaning.
+
+.. tip::
+
+    Good questions avoid judgement and avoid assumptions about the author's perspective.
+
+    Maybe you can ask for clarification? Suggest an alternative?
+    Or provide a simple explanation *why* you disagree with their proposal.
+
+    * ``This looks wrong. Are you sure it's correct?`` (eg. typo/syntax error)
+
+    * ``What do you think of "RequestFactory" instead of RequestCreator?``
+
+Even if something *is* really wrong or "a bad idea", stay respectful and
+don't get into endless you-are-wrong discussions or "flame wars".
+
+Don't use hyperbole ("always", "never", "endlessly", "nothing", "worst", "horrible", "terrible").
+
+  **Don't:** ``I don't like how you wrote this code`` - there is no clear explanation why you
```

Comment:
I am not entirely sure if using double backticks here is the best idea. To me this looks a bit weird (you can see it on the platform.sh build linked below). Maybe the 
italic would fit better?

Reply:
Another remark here: Is it intended that these paragraphs will be rendered as blockquotes?

Reply:
No they shoudn't be blockquotes, in Markdown you can 1 or 2 space indent lists if they are not a root list but Restdoc this is not supported. I discovered this while working on my own documentation wondering where these blockquotes came from 😸

Reply:
Done. Also fixed a small grammar error in the suggestion (question mark works similar to a period).